backend-processing.ts

```diff
@@ -335,6 +353,36 @@ export default class BackendProcessing {
     return database.retrieveCurrentEmployeesOfCompany(companyID);
   }
 
+  //Assumes that OriginalFundPosition table will not have duplicate combinations of IndividualID and CompanyID
+  //If this assumption is wrong, this function may return duplicate entries
+  retrievePeopleFromOriginalCompany(companyID): Promise<Object> {
```

Comment:
Same idea with the database logic in here. Also later after submitting the MVP we can refactor these functions to use await instead to make it more readable.